example/fir_filter.dart

```diff
@@ -1,115 +1,129 @@
+/// Copyright (C) 2022-2023 Intel Corporation
```

Comment:
The majority of this code was written by @wswongat who chose to omit a copyright statement.  It doesn't seem right to add an Intel copyright onto his contribution after the fact unless it also includes him as one of the authors.

Reply:
I'll fix it. I thought the Intel copyright applied to any contribution anyway, either explicitly or implicitly (for those cases where it is not specified in the code).

Reply:
Although yes, there may be times when this is not the case. For example, the same file with an example, taken from some third-party project.